ipatests/test_integration/test_commands.py

```diff
@@ -1561,6 +1561,22 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log
+        """
+        passwd = "Secret123"
+        tasks.kinit_admin(self.master)
+        cmd = self.master.run_command(
+            [
+                'journalctl', '-t',
+                '/usr/sbin/ipa-server-install', '-n1',
+                '-o', 'json-pretty', '|', 'jq', '.CMDLINE'
+            ]
+        )
+        assert passwd not in cmd.stdout_text
```

Comment:
I would collect the output of journalctl command only and the assert the passwd in it
```
cmd = self.master.run_command(
            [
                'journalctl', '-t',
                '/usr/sbin/ipa-server-install', '-n1',
                '-o', 'json-pretty'
            ]
        )
assert self.master.config.admin_password not in stdout_text
```

Also does `kinit` needed here?

Reply:
@mrizwan93 removed the kinit statement and kept the journalctl command.